From 31b18a959ab5f1822e2bd86a9e8527c2f95e706f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ludger=20Kr=C3=A4mer?= <dbluelle@onlinehome.de>
Date: Fri, 15 Nov 2024 17:44:48 +0100
Subject: [PATCH] add dummy file for nasm detection

this is a try to fix compilation with gcc in github actions
---
 conf/CMakeASM-NASMLinkerInformation.cmake | 3 +++
 1 file changed, 3 insertions(+)
 create mode 100644 conf/CMakeASM-NASMLinkerInformation.cmake

diff --git a/conf/CMakeASM-NASMLinkerInformation.cmake b/conf/CMakeASM-NASMLinkerInformation.cmake
new file mode 100644
index 0000000000..05f89421a1
--- /dev/null
+++ b/conf/CMakeASM-NASMLinkerInformation.cmake
@@ -0,0 +1,3 @@
+# support for the nasm assembler
+
+INCLUDE(CMakeASMLinkerInformation)
